aws-ssm-ec2-proxy-command-start.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env sh
+######## Source ################################################################
+#
+# https://github.com/qoomon/aws-ssm-ec2-proxy-command
+#
+######## Usage #################################################################
+# https://github.com/qoomon/aws-ssm-ec2-proxy-command/blob/master/README.md
+#
+# Install Proxy Command
+#   - Move this script to ~/.ssh/aws-ssm-ec2-proxy-command.sh
```

Comment:
I think this is misleading because this is not referring to this very own script itself

Reply:
Just describe what is need in addition to ~/.ssh/aws-ssm-ec2-proxy-command.sh
For usage steps of ~/.ssh/aws-ssm-ec2-proxy-command.sh refer to the usage steps of ~/.ssh/aws-ssm-ec2-proxy-command.sh

Reply:
adjusted :)